addons/binding/org.openhab.binding.zoneminder/build.properties

```diff
@@ -0,0 +1,7 @@
+source.. = src/main/java/
+output.. = target/classes
+bin.includes = META-INF/,\
+               .,\
+               OSGI-INF/,\
+               ESH-INF/, \
+               lib/zoneminder4j-0.1-jar-with-dependencies.jar
```

Comment:
This file is HUGE and it contains a lot of what is already available in the runtime.
You should try to use only the core library of zoneminder4j (btw, where does that live?) and add only dependencies that are really still needed.